### PR TITLE
chore: remove failing steps in lint workflow

### PR DESCRIPTION
**Pull request by tenshiAMD**
**What**:
- This step always fails so we need to remove it. 
- Got broken in https://github.com/all-contributors/all-contributors/pull/640.

**Why**:
- This step always fails so we need to remove it.

**How**:
- Remove affected line in codebase

**Checklist**:

- [x] Documentation
- [x] Ready to be merged 
- [x] Added myself to contributors table. 
[Bot Usage](https://allcontributors.org/docs/en/bot/installation#4-update-your-contributing-documentation)

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -14,7 +14,6 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: xt0rted/markdownlint-problem-matcher@78410d0625df50bd14b8e178749fdb66e07c31e0
       - uses: actions/setup-node@v3
       - run: npm install
       - run: ./node_modules/.bin/markdownlint **/*.md --ignore node_modules
```
Comment (gr2m) [inline .github/workflows/lint.yml:19]: looksl ike this line is failing? If we don't install `markdownlint` we can run the command with `npx markdownlint **/*.md --ignore node_modules`

Comment (gr2m) [reply .github/workflows/lint.yml:19]: try this

```
npx --package markdownlint-cli markdownlint **/*.md --ignore node_modules
```

**Comment by tenshiAMD** [comment]
@gr2m gentle ping. thanks! 🎉

**Comment by tenshiAMD** [comment]
> can you address the lint errors?

@gr2m sure let me check. Looks like those errors are many, this could take a while.

**Comment by gr2m** [comment]
try running the CLI locally with `--fix`, see https://github.com/igorshubovych/markdownlint-cli#usage. That might address most of the linting errors

**Comment by tenshiAMD** [comment]
> try running the CLI locally with `--fix`, see [igorshubovych/markdownlint-cli#usage](https://github.com/igorshubovych/markdownlint-cli#usage). That might address most of the linting errors

@gr2m done. thanks for the guidance 😄

**Comment by tenshiAMD** [comment]
@gr2m gentle ping

**Review by Berkmann18 — APPROVED** [review]
That looks good to me.

**Pull request merged**